src/sentry/static/sentry/app/components/alertMessage.jsx

```diff
@@ -27,11 +27,6 @@ const StyledCloseButton = styled.button`
   right: ${p => p.theme.grid}px;
   top: 7px;
 
-  /* stylelint-disable-next-line no-duplicate-selectors */
```

Comment:
this inherits the text color in the svg which is better than always setting it to gray (which looks weird)